charts/vc-authn-oidc/templates/deployment.yaml

```diff
@@ -54,36 +55,46 @@ spec:
                   key: mongodb-passwords
             - name: CONTROLLER_URL
               value: {{ include "vc-authn-oidc.url" . }}
+            - name: CONTROLLER_API_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "vc-authn-oidc.apiSecretName" . }}
+                  key: controllerApiKey
+            - name: CONTROLLER_CAMERA_REDIRECT_URL
+              value: {{ .Values.controllerCameraRedirectUrl }}
+            - name: CONTROLLER_PRESENTATION_EXPIRE_TIME
+              value: {{ .Values.controllerPresentationExpireTime | quote }}
             - name: ACAPY_AGENT_URL
-              value: {{ .Values.acaPy.agentUrl }}
+              value: https://{{ include "acapy.host" . }}
             - name: ACAPY_ADMIN_URL
-              value: {{ .Values.acaPy.adminUrl }}
+              value: https://{{ include "acapy.admin.host" . }}
```

Comment:
Same comment as before about hard-coding http scheme for URLs.

Reply:
Is it safe to assume that if the agent endpoint is using `https://` then the admin url is also using `https://`?

Reply:
I would assume both endpoints will be using the same http scheme

Reply:
Updated `ACAPY_ADMIN_URL` to use internal admin url which points to agent's service and admin port.